### ISSUE TITLE
make check: undefined reference to `pthread_getspecific'

### ISSUE THREAD

**Issue by sjackman** [bcgsc/abyss#71]
```
❯❯❯ make check
…
$ /gsc/btl/linuxbrew/bin/g++-5 -Wall -Wextra -Wno-unknown-pragmas -Os -w -pipe -march=core2  -L/gsc/btl/linuxbrew/lib -Wl,--dynamic-linker=/gsc/btl/linuxbrew/opt/glibc/lib/ld-linux-x86-64.so.2 -Wl,-rpath,/gsc/btl/linuxbrew/lib -o PairedDBG_Dinuc PairedDBG/PairedDBG_Dinuc-DinucTest.o ../PairedDBG/libpaireddbg.a ../DataLayer/libdatalayer.a ../Common/libcommon.a ../lib/gtest-1.7.0/libgtest_main.a -ldl -lm
../lib/gtest-1.7.0/libgtest_main.a(libgtest_main_a-gtest-all.o): In function `testing::internal::ThreadLocal<testing::TestPartResultReporterInterface*>::~ThreadLocal()':
gtest-all.cc:(.text._ZN7testing8internal11ThreadLocalIPNS_31TestPartResultReporterInterfaceEED2Ev[_ZN7testing8internal11ThreadLocalIPNS_31TestPartResultReporterInterfaceEED5Ev]+0xb): undefined reference to `pthread_getspecific'
```

**Comment by sjackman** [comment]
This line is incorrect:
https://github.com/bcgsc/abyss/blob/master/Unittest/Makefile.am#L191
```
PairedDBG_Dinuc_CXXFLAGS = $(AM_CXXFLAGS) $(OPENMP_CXXFLAGS)
```
should be
```
PairedDBG_Dinuc_CXXFLAGS = $(GTEST_CXXFLAGS_) $(OPENMP_CXXFLAGS)
```

**Comment by benvvalk** [comment]
Argh.  Just fixed in 62b16ea07020fab506ddaad97f5c4766ac6bc4aa.

**Comment by sjackman** [comment]
Hi, Ben. Can you add `Closes bcgsc/abyss#71` to the commit log of 62b16ea07020fab506ddaad97f5c4766ac6bc4aa and force push it?

**Comment by sjackman** [comment]
It should be possible to get rid of all the separate `*_CXXFLAGS` definitions that are all the same and replace them all with something like
```
AM_CXXFLAGS +=  $(PTHREAD_CFLAGS) -Wno-error
```
